src/pages/landing/deriv-go/components/_side-tab.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled, { css, keyframes } from 'styled-components'
+import { Flex, Show } from 'components/containers'
+import { Text } from 'components/elements'
+import device from 'themes/device'
+
+const TabContent = styled.div`
+    flex: 1;
+    width: 100%;
+
+    & .content-wrapper {
+        width: 287px;
+        height: 576px;
+        @media ${device.tabletL} {
+            width: 192px;
+            height: 386px;
+        }
+    }
+`
+
+const TabButton = styled(Flex)`
+    position: relative;
+    z-index: 2;
+    flex-direction: column;
+    justify-content: center;
+    padding-top: 40px;
+    width: 428px;
+    cursor: pointer;
+    margin-left: 9rem;
+
+    @media ${device.tabletL} {
+        width: fit-content;
+        padding-top: 10px;
+        margin-left: 0;
+    }
+`
+
+const TabList = styled.div`
+    max-width: 100%;
+    ${(props) =>
+        props.is_reverse
+            ? css`
+                  margin-left: 2.4rem;
+              `
+            : css`
+                  margin-right: 2.4rem;
+              `}
+`
+
+const TabListWrapper = styled.div`
+    max-width: 28.2rem;
+    display: flex;
+    flex-direction: column;
+
+    @media ${device.tabletL} {
+        max-width: 100%;
+        margin: 0;
+    }
+`
+const animateTab = keyframes`
+    0% {opacity:0;}
+    100% {opacity:1;}
+
+`
+
+const TextLabel = styled(Text)`
+    color: ${(props) => (props.selected ? 'rgba(51, 51, 51, 1)' : 'rgba(153, 153, 153, 1)')};
+    font-size: 32px;
+    animation: ${animateTab} 2s;
+
+    @media ${device.tabletL} {
+        font-size: 24px;
+    }
+`
+
+const TextDesc = styled(Text)`
+    display: ${(props) => (!props.selected ? 'none' : '')};
+    font-size: 24px;
+    animation: ${animateTab} 2s;
+
+    @media ${device.tabletL} {
+        font-size: 18px;
+    }
+`
+
+const Content = styled(Flex)`
+    display: grid;
+    justify-content: center;
+    padding-left: 16rem;
+    padding-top: 60px;
+    @media ${device.tabletL} {
+        padding-top: 24px;
+        padding-left: 0;
+    }
+`
+
+const CarouselDot = styled.div`
+    height: 12px;
+    align-self: center;
+    width: 12px;
+    border-radius: 50%;
+    border: ${(props) => (props.selected ? 'none' : '1px solid #000000')};
+    margin-bottom: 16px;
+    background-color: ${(props) =>
+        props.selected ? 'rgba(255, 68, 79, 1)' : 'rgba(248, 250, 251, 1)'};
+    @media ${device.tabletL} {
+        display: none;
+    }
+`
+const CarouselContainer = styled.div`
+    height: 80px;
+    align-self: center;
+`
```

Comment:
```suggestion
    height: 80px;
    align-self: center;
    margin-right: 36px;
`
```

Reply:
You can introduce a margin-right here so that the dots won't ever touch the right side of the viewport when in view.

Reply:
sure, I will